rma/tests/test_rma.py

```diff
@@ -848,3 +849,67 @@ def test_autoconfirm_email(self):
         )
         self.assertTrue(rma.name in mail_receipt.subject)
         self.assertTrue("products received" in mail_receipt.subject)
+
+    def _test_receive_and_replace(self):
```

Comment:
I don't feel comfortable with helper methods starting with `_test`, as they seem "commented" methods to not be run, and more if they are not doing any assert.